pom.xml

```diff
@@ -780,10 +787,24 @@
           <exists>${basedir}/src/doc/javascript/config/${docjs.config.file}</exists>
         </file>
       </activation>
+
       <properties>
         <jsdoc3-maven-plugin.version>1.2.0</jsdoc3-maven-plugin.version>
         <docjs.config.includePrivate>false</docjs.config.includePrivate>
+
+        <dependency.jsdoc.template.version>${project.version}</dependency.jsdoc.template.version>
+        <dependency.jsdoc.template.artifactId>common-ui-jsdoc-template</dependency.jsdoc.template.artifactId>
       </properties>
+
+      <dependencies>
+        <dependency>
+          <groupId>org.pentaho</groupId>
+          <artifactId>${dependency.jsdoc.template.artifactId}</artifactId>
```

Comment:
I don't think using a property to define the template artifact to use is good pratice.